setup.cfg

```
@@ -15,7 +15,7 @@ classifiers =
 [options]
 install_requires =
     fsspec>=2021.8.1
-    asyncssh>=2.11.0,<3
+    asyncssh==2.19.0
```

Let's keep it this way:

```suggestion
    asyncssh>=2.11.0,<3
```